plugins/modules/role_info.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Paul Arthur <paul.arthur@flowerysong.com>
+# Copyright: (c) 2019, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: role_info
+author:
+  - Paul Arthur (@flowerysong)
+  - Manca Bizjak (@mancabizjak)
+  - Aljaz Kosir (@aljazkosir)
+  - Tadej Borovsak (@tadeboro)
+short_description: Lists Sensu roles
+description:
+  - For more information, refer to the Sensu documentation at
+    U(https://docs.sensu.io/sensu-go/latest/reference/roles/)
```

Comment:
404 page not found :smiley:, I think you meant `https://docs.sensu.io/sensu-go/latest/reference/rbac/`

Reply:
:scream: Fixed